gadgets/advise_seccomp/gadget.yaml

```diff
@@ -0,0 +1,14 @@
+name: advise seccomp
+description: Suggest a seccomp profile
+homepageURL: https://inspektor-gadget.io/
+documentationURL: https://www.inspektor-gadget.io/docs/latest/gadgets/advise_seccomp
+sourceURL: https://github.com/inspektor-gadget/inspektor-gadget/tree/main/gadgets/advise_seccomp
+datasources:
+  syscalls:
+    annotations:
+      cli.output-mode: none
+      ebpf.map.flush-on-stop: true
+  advise:
+    annotations:
+      cli.supported-output-modes: advise
+      cli.default-output-mode: advise
```

Comment:
`advise` is not listed as possible output-mode in docs/spec/operators/cli.md

Reply:
This output mode is specific for this gadget, so it shouldn't be documented on the operator.

Reply:
I didn't know a gadget could add a custom output mode. Should the operator documentation say that a gadget can implement its own?

Reply:
Done in the last commit.